src/Microsoft.Data/DataFrame.cs

```diff
@@ -350,13 +385,7 @@ public DataFrame Description()
                 }
                 DataFrame columnDescription = column.Description();
                 ret = ret.Merge<string>(columnDescription, "Description", "Description", "_left", "_right", JoinAlgorithm.Inner);
-                int leftMergeColumn = ret._table.GetColumnIndex("Description" + "_left");
-                int rightMergeColumn = ret._table.GetColumnIndex("Description" + "_right");
-                if (leftMergeColumn != -1 && rightMergeColumn != -1)
-                {
-                    ret.RemoveColumn("Description" + "_right");
-                    ret._table.SetColumnName(ret["Description_left"], "Description");
-                }
+                RemoveRightAndRenameLeftColumn(ret, "Description", "_left", "_right");
```

Comment:
Does this need to be changed anymore?
Also, could we follow the same pattern here as `Info`, where the DataFrame is passed into the `Description()` method? Then we wouldn't have to merge and unmangle the resultant DataFrames.